bower.json

```diff
@@ -40,7 +40,8 @@
         "MediaUploader": "dimagi/MediaUploader#master",
         "xpath": "dimagi/js-xpath#v0.0.3",
         "jsdiff": ">=1",
-        "markdown-it": "~6.0.0"
+        "markdown-it": "~6.0.0",
+        "codemirror": "3.24.0"
```

Comment:
HQ uses 5.5.0, but https://github.com/dimagi/Vellum/blob/master/lib/codemirror/xml.js is not able to run that version (we don't use HQ's version)